easybuild/easyblocks/s/superlu.py

```diff
@@ -26,6 +26,9 @@
 EasyBuild support for building and installing the SuperLU library, implemented as an easyblock
 
 @author: Xavier Besseron (University of Luxembourg)
+
+Updated to use FlexiBlas, updated for version 5.3.0, come cleanup
+Author: J. Sassmannshausen (ICL/UK)
```

Comment:
There's usually no comment about what an author contributed, it's just a (single-entry) time-ordered list
```suggestion
@author: J. Sassmannshausen (ICL/UK)
```